app/models/queue_it/queue.rb

```diff
@@ -88,8 +88,7 @@ def push_in_tail(nodable)
         lock!
         old_tail_node = tail_node.lock!
         old_tail_node.update!(kind: :any)
-        new_tail_node = nodes.create!(nodable: nodable, kind: :tail)
-        old_tail_node.update!(parent_node: new_tail_node)
+        new_tail_node = nodes.create!(nodable: nodable, kind: :tail, parent_node: old_tail_node)
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Lint/UselessAssignment: Useless assignment to variable - new_tail_node. Did you mean old_tail_node?